app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> str:
+    parts = command.split()
```

Comment:
You could use unpacking and avoid indexing to improve the readability
```suggestion
    command_name, source_file, destination_file, *_ = command.split()
```